integration-templates/lever/lever-create-note.ts

```diff
@@ -0,0 +1,63 @@
+import type { NangoAction, LeverOpportunityNote } from './models';
+
+interface LeverCreateNoteInput {
+    opportunityId: string;
+    perform_as?: string;
+    note_id?: string;
+    value: string;
+    secret?: boolean;
+    score?: number;
+    notifyFollowers?: boolean;
+    createdAt?: number;
+}
+
+export default async function runAction(nango: NangoAction, input: LeverCreateNoteInput): Promise<LeverOpportunityNote> {
+    if (!input.opportunityId) {
+        throw new nango.ActionError({
+            message: 'opportunity id is a required field'
+        });
+    } else if (!input.value) {
+        throw new nango.ActionError({
+            message: 'value of the note is a required field'
+        });
+    }
+
+    const endpoint = `/v1/opportunities/${input.opportunityId}/notes`;
+
+    try {
+        const postData = {
+            value: input.value,
+            secret: input.secret,
+            score: input.score,
+            notifyFollowers: input.notifyFollowers,
+            createdAt: input.createdAt
+        };
+
+        const params = Object.entries({
+            ...(input.perform_as ? { perform_as: input.perform_as } : {}),
+            ...(input.note_id ? { note_id: input.note_id } : {})
+        })
+            .map(([key, value]) => `${encodeURIComponent(key)}=${encodeURIComponent(value)}`)
+            .join('&');
+
+        const urlWithParams = `${endpoint}${params ? `?${params}` : ''}`;
+
+        const resp = await nango.post({
+            endpoint: urlWithParams,
+            data: postData
+        });
+
+        return {
+            id: resp.data.data.id,
```

Comment:
is `data.data` expected?

Reply:
Yes, the response has a nested `data` property within the `data` property. 
![notes](https://github.com/NangoHQ/nango/assets/85742599/b551645b-3c37-4136-b422-b18e39c86d4e)